src/scripts/dice.js

```diff
@@ -11,6 +11,7 @@ const randomRoll = () => {
 };
 
 const animateDie = (random) => {
+    dice = document.querySelector('.dice');
```

Comment:
Why are do you query the dice twice if you its already a global variable?

Reply:
Side effect of the structural changes on the pages, would be better to revert.